engine/components-api/src/main/java/com/cloud/ha/HighAvailabilityManager.java

```diff
@@ -28,6 +30,9 @@
  * HighAvailabilityManager checks to make sure the VMs are running fine.
  */
 public interface HighAvailabilityManager extends Manager {
+
```

Comment:
@GabrielBrascher why do I see java changes in this PR?

Reply:
My bad @DaanHoogland, I was debugging this and did commit by mistake, will soon be removed soon.